google_helpers/stackdriver.py

```diff
@@ -97,12 +71,14 @@ def write_text_log_entry(self, log_name, log_text, severity="DEFAULT" ):
             'textPayload': log_text
         }])
 
+
+    # This *IS* used in a few places 4/25/25. Converting to ISB-CGC form!
     @classmethod
     def build_from_django_settings(cls):
         from django.conf import settings
+        #This is from ISB-CGC version
+        #project_name = settings.GCLOUD_PROJECT_ID
+        # This is from IDC version:
         project_name = settings.BIGQUERY_PROJECT_ID
-        credentials = ServiceAccountCredentials.from_json_keyfile_name(
-            settings.GOOGLE_APPLICATION_CREDENTIALS, cls.LOGGING_SCOPES)
-
-        return cls(project_name, credentials)
+        return cls(project_name)
```

Comment:
@wlongabaugh Is there a constructor that doesn't need credentials now? It doesn't look like it.

Reply:
Fixed